assembly/__tests__/vm-mock.spec.ts

```diff
@@ -64,4 +64,11 @@ describe('Testing mocked Storage and CallStack', () => {
   test('Testing event', () => {
     generateEvent("I'm an event ");
   });
+
+  test('Testing sha256', () => {
+    const result = bytesToString(sha256(stringToBytes('something')));
+    expect(result).toBe(
+      '3fc9b689459d738f8c88a3a48aa9e33542016b7a4052e001aaa536fca74813cb',
```

Comment:
Are you sure that the blockchain is using the same case ? '3f...' and not '3F...'

Reply:
@AurelienFT ? 

In any case here we are testing the mock so it's not really serious